src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -431,7 +431,7 @@ def reconstruct_messages(
                     "timestamp": old_payload_value["timestamp"],
                     "project_id": old_payload_value["project_id"],
                     "type": old_payload_value["type"],
-                    "value": old_payload_value["value"],
+                    "value": old_payload_value["value"],  # type: ignore
```

Comment:
type ignore should only be used for bugs in the type checker -- please fix the actual problem here instead of ignoring it

Reply:
Hi, the fix here would be to change the Kafka schema [here](https://github.com/getsentry/sentry-kafka-schemas/blob/main/schemas/ingest-metrics.v1.schema.json) to include a value type it doesn't actually support, which may not make a lot of sense. 

The reason why we are in this state is because: the way the schemas are set up right now is, there are two separate pipelines sharing one Kafka schema upstream, but having their own Kafka schemas downstream. The upstream schema supports this new value, but only 1 of the 2 downstream schemas supports the new value. We would need to fully refactor the upstream schema and split it into 2 schemas in order to get this to work as it should.